app/Http/routes.php

```diff
@@ -10,7 +10,69 @@
 | and give it the controller to call when that URI is requested.
 |
 */
-
+//user
 Route::get('/', function () {
-    return view('welcome');
+    return view('pages.user.main');
+});
+Route::get('/Contact', function () {
```

Comment:
route name should not use `Camel` style, please use dash style. 
- `/contact`
- `/user-profile`